scripts/debug_tools/renew_info_files.py

```diff
@@ -0,0 +1,236 @@
+                   (b) unparsed target (string),
+                   (c) default compiler standard (string).
+        filepath : Path to 'compiler_info.json' file that should be created.
+    """
+    info = dict()
+
+    for compiler in old_info:
+        include_paths = process_includes(old_info[compiler]['includes'])
+        for idx, _ in enumerate(include_paths):
+            include_paths[idx] = "-isystem %s" % include_paths[idx]
+        info[compiler] = {
+            "includes": include_paths,
+            "target": process_target(old_info[compiler]['target']),
+            "default_standard": old_info[compiler]['default_standard']}
+
+    with io.open(filepath, 'w', encoding='UTF-8') as dest:
+        dest.write(str(json.dumps(info), 'utf-8'))
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+            description="Convert old compiler info or even older compiler "
+                        "target and includes files into one info file that "
+                        "can be used with new CodeChecker versions.")
+    parser.add_argument('dir', metavar='DIR', help='Path to directory where '
+                        'the old-version "compiler_info.json" or the pair of '
+                        '"compiler_[includes/target].json" files are located.')
+    parser.add_argument('--clean', action='store_true', required=False,
+                        default=False, help='remove old '
+                        '"compiler_[includes/target/info].json" files')
+    parser.add_argument('--verbose', action='store_true', required=False,
+                        default=False, help='print debug information')
+    args = parser.parse_args()
+
+    if not os.path.isdir(args.dir):
+        sys.stderr.write("%s is not a directory\n" % args.dir)
+        sys.exit(1)
+
+    target_file = os.path.join(args.dir, 'compiler_target.json')
+    info_file = os.path.join(args.dir, 'compiler_info.json')
+
+    if not os.path.isfile(info_file) and not os.path.isfile(target_file):
+        sys.stderr.write("Neither an old-version 'compiler_info.json' "
+                         "nor a 'compiler_target.json' could be found in %s\n"
+                         % args.dir)
+        sys.exit(2)
+
+    if os.path.isfile(info_file):
+        # We have found a 'compiler_info.json' file.
+        with io.open(info_file, 'r', encoding='UTF-8') as src:
+            info = json.loads(src.read())
+
+        if not old_version_detected(info, args.verbose):
+            # If the info file is not an old-version one, we are not sure
+            # what to do with it. Notify the user and exit.
+            sys.stderr.write("'compiler_info.json' already exists!\n")
+            sys.exit(3)
+
+        if not args.clean:
+            # Rename the old JSON, otherwise we will overwrite it.
+            os.rename(info_file,
+                      os.path.join(args.dir, 'compiler_info_old.json'))
+            if args.verbose:
```

Comment:
Wouldn't it be better to use python logger instead of checking always that `verbose` option is enabled if you would like to print a debug message?

Similar to this: https://github.com/Ericsson/codechecker/blob/3970836c02a49b8e1851aa81e25d2f12533de003/scripts/build_package.py#L31-L37

What do you think @gyorb

Reply:
Yes using the logging library has some benefits.